### DISCUSSION TITLE
Breakpoint não funciona

### DISCUSSION THREAD

**Discussion by claudiavieira** [balta-io/forum#681] [Tira Dúvidas]
Gostaria de utilizar alguns breakpoints para debugar um erro, entretanto, não funcionam os breakpoints.
Estou executando o comando "dotnet run" tanto no projeto Dima.Api quanto Dima.Web. 

Curso: Fullstack .NET

**Comment by henriqueweege** [comment]
Oi, @claudiavieira ! 
Para usar o comando e conseguir debugar é necessário [fazer o "attach" do processo a sua IDE](https://blog.devgenius.io/debug-with-dotnet-run-cli-c70dcdc98237). Para evitar esse trabalho, é mais fácil usar as "setinhas" para rodar direto da IDE em modo de depuração. 
Se tiver problema encontrando as da sua IDE, só avisa aqui qual IDE vc está usando que desenrolo pra você, bele?

Valeeeu!

**Reply by claudiavieira** [reply]
Olá Henrique, obrigada por sua ajuda. 
Ao debugar via prompt de comando, (dotnet run),  consigo efetuar o login no projeto, porém não consigo debugar.  

Ao tentar via IDE,  os breakpoints funcionam, mas recebo um retorno com o erro "TypeError: Failed to fetch".
Deixei os projetos da API e Web iniciados, conforme figura abaixo.
![1](https://github.com/user-attachments/assets/402444c6-0eff-4fa7-ae32-fb7eca04080c)

**Reply by henriqueweege** [reply]
Oi, @claudiavieira !
Boa! Esse já é outro erro, e erros novos geralmente são bom sinal com software 😄 
Esse novo erro parece estar relacionado com CORS. Teria como dar uma olhada se [esse post do StackOverflow](https://stackoverflow.com/questions/77098574/blazor-webassembly-how-to-fix-typeerror-failed-to-fetch) te ajuda?

Valeeeu!

**Comment by atilionogueira** [comment]
@henriqueweege, bom dia! muito boa a sua dica para fazer debug
